src/processor/model/otel_transform.rs

```diff
@@ -0,0 +1,127 @@
+use crate::processor::model::Value::{BoolValue, BytesValue, DoubleValue, IntValue, StringValue};
+use crate::processor::model::{ResourceSpans, Span};
+use std::sync::{Arc, Mutex};
+
+pub fn transform(rs: opentelemetry_proto::tonic::trace::v1::ResourceSpans) -> ResourceSpans {
+    let mut resource_span = ResourceSpans {
+        resource: Arc::new(Mutex::new(None)),
+        scope_spans: Arc::new(Mutex::new(vec![])),
+        _schema_url: Arc::new(Mutex::new("".to_string())),
+    };
+    if rs.resource.is_some() {
+        let resource = rs.resource.unwrap();
+        let kvs = build_rotel_sdk_resource(resource);
+        let res = Arc::new(Mutex::new(Some(crate::processor::model::Resource {
+            attributes: Arc::new(Mutex::new(kvs.clone())),
+            dropped_attributes_count: Arc::new(Mutex::new(0)),
```

Comment:
Does this need to copy from resource.dropped_attributes_count?

Reply:
Yeah we need a TODO there, added elsewhere but missed that one